trunk/SUAVE/Methods/Propulsion/serial_HTS_dynamo_turboelectric_sizing.py

```diff
@@ -0,0 +1,284 @@
+
+
+
+
+## @ingroup Methods-Propulsion
+def serial_HTS_dynamo_turboelectric_sizing(Turboelectric_HTS_Dynamo_Ducted_Fan, mach_number = None, altitude = None, delta_isa = 0, conditions = None, cryo_cold_temp = 50.0, cryo_amb_temp = 300.0):  
+    """
+    create and evaluate a serial hybrid network that follows the power flow:
+    Turboelectric Generators -> Motor Drivers -> Electric Propulsion Motors
+    where the electric motors have cryogenically cooled HTS rotors that follow the power flow:
+    Turboelectric Generators -> HTS Dynamo -> HTS Rotor Coils
+    and
+    Turboelectric Generators -> Cryocooler <- HTS Rotor Heat Load
+    There is also the capability for the HTS components to be cryogenically cooled using liquid or gaseous cryogen, howver this is not sized other than applying a factor to the cryocooler required power.
+
+    creates and evaluates a ducted_fan network based on an atmospheric sizing condition
+    creates and evaluates a serial hybrid network that includes a HTS motor driven ducted fan, turboelectric generator, and the required supporting equipment including cryogenic cooling and HTS dynamo current supply.
+
+    Assumptions:
+        One powertrain model represents all engines in the model.
+        There are no transmission losses between components
+        the shaft torque and power required from the fan is the same as what would be required from the fan of a turbofan engine.
+
+    Source:
+        N/A
+
+    Inputs:
+        Turboelectric_HTS_Dynamo_Ducted_Fan    Serial HTYS hybrid ducted fan network object (to be modified)
+        mach_number
+        altitude                        [meters]
+        delta_isa                       temperature difference [K]
+        conditions                      ordered dict object
+
+    Outputs:
+        N/A
+
+    Properties Used:
+        N/A
+    """
+    
+    # Unpack components
+    ducted_fan      = Turboelectric_HTS_Dynamo_Ducted_Fan.ducted_fan       # Propulsion fans
+    motor           = Turboelectric_HTS_Dynamo_Ducted_Fan.motor            # Propulsion fan motors
+    turboelectric   = Turboelectric_HTS_Dynamo_Ducted_Fan.powersupply      # Electricity providers
+    esc             = Turboelectric_HTS_Dynamo_Ducted_Fan.esc              # Propulsion motor speed controllers
+    rotor           = Turboelectric_HTS_Dynamo_Ducted_Fan.rotor            # Propulsion motor HTS rotors
+    hts_dynamo      = Turboelectric_HTS_Dynamo_Ducted_Fan.hts_dynamo       # HTS Dynamo current supply
+    dynamo_esc      = Turboelectric_HTS_Dynamo_Ducted_Fan.dynamo_esc       # HTS Dynamo speed controller
+    cryocooler      = Turboelectric_HTS_Dynamo_Ducted_Fan.cryocooler       # HTS rotor cryocoolers
+    heat_exchanger  = Turboelectric_HTS_Dynamo_Ducted_Fan.heat_exchanger   # HTS rotor cryocooling via cryogen
+
+    # Dummy values for specifications not currently used for analysis
+    motor_current   = 100.0
```

Comment:
I started looking at the [HTS_DC_Supply](https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Components/Energy/Distributors/HTS_DC_Supply.py) and it doesn't follow the typical SUAVE structure. It should have a common interface of conditions and the inputs should be applied to the object. That would remove the requirement of keeping current in the interface.